lib/shape.lua

```diff
@@ -169,6 +172,42 @@ function Shape:tick()
 	while self.theta > tau do
 		self.theta = self.theta - tau
 	end
+	while self.theta < -tau do
+		self.theta = self.theta + tau
+	end
```

Comment:
This check was missing and so counter-clockwise turning shapes kept increasing theta indefinitely, which made my ARC 3 LED update bug out.